ansible/roles/jenkins-workspace/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+#
+# Create a Git server to be used for CI temporary binaries
+# https://git-scm.com/book/en/v2/Git-on-the-Server-Setting-Up-the-Server
+#
+
+- name: Run which git-shell
+  command: "which git-shell"
+  register: which_git_shell
+
+- name: Add git-shell to /etc/shells
+  when: "which_git_shell.rc == 0"
+  lineinfile:
+    dest: "/etc/shells"
+    state: present
+    line: "{{ which_git_shell.stdout }}"
+
+- name: Create binary_tmp group
+  group:
+    name: "binary_tmp"
+
+- name: Create binary_tmp user
+  user:
+    name: "binary_tmp"
+    group: "binary_tmp"
+    shell: "{{ which_git_shell.stdout }}"
+
+- name: Download GitHub pubkey for nodejs-ci
+  get_url:
+    url: "https://github.com/nodejs-ci.keys"
+    dest: "/tmp/nodejs-ci.keys"
+  delegate_to: 127.0.0.1
+  become: no
+
+- name: Add nodejs-ci to authorized_keys for binary_tmp
+  authorized_key:
+    user: "binary_tmp"
+    key: "{{ lookup('file', '/tmp/nodejs-ci.keys') }}"
+
+# Repository needs to be created in /home/iojs/build because the partition with
+# free space might be mounted in a way that does not include /home/binary_tmp
+- name: Create repository directory
+  file:
+    path: "~iojs/build/binary_tmp.git"
```

Comment:
better be explicit and use `{{ server_user }}` like everywhere else - `/home/{{ server_user }}/build/binary_tmp.git`